contracts/linear/src/validator_pool.rs

```diff
@@ -13,6 +13,7 @@ use std::cmp::min;
 
 const STAKE_SMALL_CHANGE_AMOUNT: Balance = ONE_NEAR;
 const UNSTAKE_FACTOR: u128 = 2;
+const MAX_SYNC_BALANCE_DIFF: Balance = 100;
```

Comment:
How do we come up with the `100yN` diff constant here?

Reply:
I made up this number by observing some validators on testnet, it could be adjusted later

Reply:
sounds good.